README.md

```diff
@@ -1,3 +1,226 @@
-# jboss-kie-modules
-JBoss KIE Modules
+# JBoss KIE Modules
 
+Here are located all CeKit modules, all the modules that will be used to configure and build the images are placed here.
+These modules are copied into the container at build time. There are also external dependencies, like jboss-eap-modules,
+cct_modules, etc.; all **external** modules can be found on the modules section of the image.yaml file descriptor for each image.
+
+Below, all CeKit modules contained in this repository:
+
+
+jboss-kie-modules \
+├── [jboss-kie-controller](jboss-kie-controller): RHDM/PAM controller specific modules. \
+├── [jboss-kie-kieserver](jboss-kie-kieserver): RHDM/PAM Execution Server specific modules. \
+├── [jboss-kie-optaweb-common](jboss-kie-optaweb-common): Optaweb common modules. \
+├── [jboss-kie-optaweb-employeerostering](jboss-kie-optaweb-employeerostering): Optaweb Employee Rostering specific modules. \
+├── [jboss-kie-smartrouter](jboss-kie-smartrouter): RHPAM Smart Router specific modules. \
+├── [jboss-kie-wildfly-common](jboss-kie-wildfly-common): Common modules, basically used by all images. \
+├── [jboss-kie-workbench](jboss-kie-workbench):  RHDM/PAM Decision/Business Central specific modules. \
+├── [jboss-kie-workbench-indexing](jboss-kie-workbench-indexing): RHDM/PAM Decision/Business Central indexing specific modules. \
+├── [os.kieserver.chmod](os.kieserver.chmod): Process/Decision Server module. \
+├── [os.kieserver.launch](os.kieserver.launch): Process/Decision Server module. \
+├── [os.kieserver.probes](os.kieserver.probes): Process/Decision liveness and readineses probes. \
+├── [os.kieserver.s2i](os.kieserver.s2i): Process/Decision Server module. \
+├── [os.kieserver.webapp](os.kieserver.webapp): Process/Decision Server module. \
+├── [tests](tests): \
+│&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;├── [arquillian](tests/arquillian): Integrations tests based on Arquillian Cube (IPS/DS 6.4.x). \
+│&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;├── [bats](tests/bats): Common files for Bats tests. \
+│&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;└── [features](tests/features): Image tests (behave) \
+└── [tools](tools) \
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;├── [build-overrides](tools/build-overrides): Tool used to help to perform image builds using nighlty artifact builds. \
```

Comment:
Tool used to help to perform image builds using nighlty, staging, or candidate artifact builds.